Kickstarter.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -5,17 +5,17 @@
       "kind" : "remoteSourceControl",
       "location" : "https://github.com/google/abseil-cpp-binary.git",
       "state" : {
-        "revision" : "a5f16ba68913840ee5df91b8dc06f5cc063579de",
-        "version" : "1.2021110200.0"
```

Comment:
Do we need to commit these version updates in this PR?

Reply:
Okay good call. I'll remove these updates because they could potentially introduce breaking changes. It seemed okay in testing, but these dependencies were not tested individually in their use cases. It's on my plate to go over the dependencies with Isabel and understand what we need to update each quarter. We'll update this file separately.

Reply:
great. thanks